templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}

```

Comment:
You should add Cancel button apart from Delete, [checklist](https://github.com/mate-academy/py-taxi-service-forms/blob/master/checklist.md)